src/vario/buttons.h

```diff
@@ -1,37 +1,50 @@
-#ifndef buttons_h
-#define buttons_h
+#pragma once
 
 #include <Arduino.h>
 
-
-//Pinout for Leaf V3.2.0
-#define BUTTON_PIN_CENTER  2  // INPUT
-#define BUTTON_PIN_LEFT    3  // INPUT
-#define BUTTON_PIN_DOWN    4  // INPUT
-#define BUTTON_PIN_UP      5  // INPUT
-#define BUTTON_PIN_RIGHT   6 // INPUT
+// Pinout for Leaf V3.2.0
+#define BUTTON_PIN_CENTER 2 // INPUT
+#define BUTTON_PIN_LEFT 3   // INPUT
+#define BUTTON_PIN_DOWN 4   // INPUT
+#define BUTTON_PIN_UP 5     // INPUT
+#define BUTTON_PIN_RIGHT 6  // INPUT
 
 /*
 //Pinout for Breadboard
-#define BUTTON_PIN_UP     40  
+#define BUTTON_PIN_UP     40
 #define BUTTON_PIN_DOWN   38
 #define BUTTON_PIN_LEFT   39
 #define BUTTON_PIN_RIGHT  42
 #define BUTTON_PIN_CENTER 41
 */
 
-enum buttons {NONE, UP, DOWN, LEFT, RIGHT, CENTER, BOUNCE};
-enum button_states {NO_STATE, PRESSED, RELEASED, HELD, HELD_LONG};
-
-uint8_t buttons_init(void);
-
-uint8_t buttons_check(void);
-uint8_t buttons_inspectPins(void);
-uint8_t buttons_debounce(uint8_t button);
-uint8_t buttons_get_state(void);
+// D pad button states.
+// NOTE:  Left is -1 as to make casting to an int for settings easier
+enum buttons : int8_t
+{
+    NONE = 0,
+    LEFT = -1,
+    RIGHT = 1,
+    UP = 2,
+    DOWN = 3,
+    CENTER = 4,
+    BOUNCE = 5
+};
+enum button_states : uint8_t
```

Comment:
```suggestion
enum button_state : uint8_t
```

(since a value of this type represents a single state)